pyproject.release.toml

```diff
@@ -0,0 +1,136 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mlflow"
+version = "2.14.2.dev0"
+description = "MLflow is an open source platform for the complete machine learning lifecycle"
+readme = "README.rst"
+keywords = ["mlflow", "ai", "databricks"]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "Intended Audience :: End Users/Desktop",
+  "Intended Audience :: Science/Research",
+  "Intended Audience :: Information Technology",
+  "Topic :: Scientific/Engineering :: Artificial Intelligence",
+  "Topic :: Software Development :: Libraries :: Python Modules",
+  "License :: OSI Approved :: Apache Software License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python :: 3.8",
+]
+requires-python = ">=3.8"
+dependencies = [
+  "mlflow-skinny==2.14.2.dev0",
```

Comment:
pip supports version control systems (VCS):

```
pandas @ git+https://github.com/pandas-dev/pandas`
```
but `mlflow-skinny` can't be installed using this format.